docs/build/build-parachains.md

```diff
@@ -332,9 +323,9 @@ If you would like to connect to a parachain via [Polkadot-JS Apps](https://polka
 you may do so by clicking on the network selection at the top left-hand corner of the navigation and
 selecting any parachain of choice.
 
-![rococo parachains](../assets/polkadotjs_network_parachains.png)
+![rococo parachains](../assets/paseo-chains.png)
 
-For the purpose of these following examples, we will be using the Rococo testnet "Custom Node"
+For the purpose of these following examples, we will be using the Paseo testnet "Custom Node"
 underneath "Development", following the
 [parachain tutorials](https://docs.substrate.io/reference/how-to-guides/parachains/connect-to-a-relay-chain/).
```

Comment:
This is outdated; we can link template to core for now, which applies to Paseo, then Polkadot Docs later?

Reply:
yes please :)